ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/event.proto

```diff
@@ -101,9 +124,17 @@ message ExercisedEvent {
   // Required
   bool consuming = 8;
 
-  // field ID 9 deprecated
-
-  // The parties that are notified of this event.
+  reserved 9; // removed field
+
+  // The parties that are notified of this event. The witnesses of an exercise
+  // node will depend on whether the exercise was consuming or not.
+  //
+  // If consuming, the witnesses are the union of the stakeholders and
+  // the actors.
+  //
+  // If not consuming, the witnesses are the union of the signatories and
+  // the actors.
```

Comment:
should we expand a bit on what "actors" are here and in what situation the actor would not be a signatory?

Reply:
Good idea, see b7e761c1348cf3e93f8b2d091258cf0cd8eb7023